### DISCUSSION TITLE
I need your help to run this codebase on my PC.

### DISCUSSION THREAD

**Discussion by tanyafeest** [naemon/naemon-core#458] [General]
Hello. Every one.
How are you?
I hope you are going well.
So I written to need help to run this code base on my pc.
I am using Ubuntu 20.4 currently. 
So How can I run this  code base on my PC.
Please help me.

**Comment by nook24** [comment]
Hi, do you like to run Naemon as it is, or do you plan to do some development?

**Run Naemon**

- In case you want to run Naemon as it is, I suggest to install the pre built packages you can find on the website: http://www.naemon.io/download/
- You can also build Naemon from source: https://statusengine.org/tutorials/install-naemon-focal/

Development environment
If wou want to setup a development environment to do some code changes, please see the readme. It contains two guids of how to set up an IDE for Naemon. I personally use the VS Code one: https://github.com/naemon/naemon-core?tab=readme-ov-file#development-environment

**Comment by tanyafeest** [comment]
Thank you for your help.
Have good weekend.
